_episodes/05-accelerating-lammps.md

```diff
@@ -0,0 +1,84 @@
+---
+title: "Introduction"
+teaching: 0
+exercises: 0
+questions:
+- "Key question (FIXME)"
+objectives:
+- "First learning objective. (FIXME)"
+keypoints:
+- "First key point. Brief Answer to questions. (FIXME)"
+---
+ # Accelerating LAMMPS using KOKKOS
+   
+## What is Kokkos?
+
```

Comment:
Agree, you need to remember that this lesson is for an instructor so that they can give the lesson, and also for a student as a reference after they have received the lesson.  We shouldn't make the assumption that the instructor actually knows what Kokkos is